scripts/cpu_percentage.sh

```diff
@@ -10,7 +10,10 @@ cpu_percentage_format="%3.1f%%"
 print_cpu_percentage() {
   cpu_percentage_format=$(get_tmux_option "@cpu_percentage_format" "$cpu_percentage_format")
 
-  if command_exists "iostat"; then
+  if [[ $(get_tmux_option @cpu_wsl_host) == "true" && -n "${WSL_DISTRO_NAME}" ]] ; then
```

Comment:
any reason not to just do this?

```suggestion
  if command_exists "wmic.exe"; then
```

or even better, alter the `is_cygwin` function & convert pre-exisiting `WMIC` calls to `wmic.exe`?

Reply:
I was trying to ensure we gave users the options if they wanted to just see the guest/WSL performance or the whole system and to make sure it only ran if you actually were inside WSL (I use the same .tmux.conf on multiple systems)

> or even better, alter the is_cygwin function & convert pre-exisiting WMIC calls to wmic.exe?

This might be a better option, although cygwin and WSL are different beasts - I'll install cygwin and give it a go when I've got chance.

Reply:
no need to install cygwin -- I mean we can change `is_cygwin` to simply `is_win` (i.e. `is_wsl_or_cygwin`)